tests/reflow.js

```diff
@@ -43,7 +42,7 @@ export default {
 
 			styleObserver.observe();
```

Comment:
Oops. It's a leftover we should probably remove. It was already called on `styleObserver` initialization. My bad. 🤦‍♂️

Reply:
Can you remove it?